pom.xml

```diff
@@ -32,6 +32,11 @@
 			<artifactId>config</artifactId>
 			<version>1.4.3</version>
 		</dependency>
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.15.2</version>
```

Comment:
There is much newer version of Jackson, 2.18.2.
You can check here: https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind

Reply:
Upgraded.